packages/contract_templates/contract.handlebars

```diff
@@ -1,7 +1,3 @@
-/**
```

Comment:
Why did you decide to remove that?

Reply:
Because this file (with template code) was not generated using abi-gen, and you should indeed edit it directly, per https://github.com/0xProject/0x-monorepo/tree/v2-prototype/packages/abi-gen#how-to-write-custom-templates .  I assumed that this contract.handlebars file was created by taking a wrapper (that was auto-generated using abi-gen) and editing it to add the template stuff, and that this comment was leftover from that process.

Reply:
Ideally, `abi-gen` will prepend it itself. Either do the change in `abi-gen` or leave it there for now and I'll change it later

Reply:
I do think it's right for abi-gen to prepend itself to the generated file.  But once you take the generated file and give it a life of its own by committing it with changes (here, the template additions), it's no longer a "generated file", and it makes sense to remove the comment from that file.  Humbly but strongly, my opinion is that leaving `abi-gen` as is, *and* removing this comment, are both the right things to do.  But I'm happy to defer to your opinion: if you disagree, then follow up *one* more time and tell me to remove it, and it shall be done. :)